aten/src/ATen/EmptyTensor.h

```diff
@@ -15,6 +15,12 @@ inline void check_size_nonnegative(IntArrayRef size) {
   }
 }
 
+inline void check_size_nonnegative(SymIntArrayRef size) {
+  // TODO: do this.  Note that naive implementation will choke on truly
+  // unknown sizes without on the fly reasoning
+  return;
```

Comment:
Actually I think we can just use the above implementation with no changes (maybe we should template it)